packages/p2p/src/components/order-details/order-details-timer.jsx

```diff
@@ -6,17 +6,17 @@ import { secondsToTimer } from 'Utils/date-time';
 import ServerTime from 'Utils/server-time';
 import { useStores } from 'Stores';
 
-const OrderDetailsTimer = observer(() => {
+const OrderDetailsTimer = () => {
     const getTimeLeft = time => {
         const distance = ServerTime.getDistanceToServerTime(time);
         return {
             distance,
-            label: distance < 0 ? localize('expired') : secondsToTimer(distance),
+            label: secondsToTimer(Math.max(0, distance)),
```

Comment:
What about showing `Expired`, is that not needed anymore?

Reply:
Not for this specific timer (for the one in orders table we still do). This timer was made consistent with the mobile app.